libs/sdk-ui-kit/src/utils/tests/featureFlags.test.ts

```diff
@@ -0,0 +1,126 @@
+// (C) 2020-2021 GoodData Corporation
+
+import { isFreemiumEdition, shouldEnableNewNavigation, shouldHidePPExperience } from "../featureFlags";
+
+describe("featureFlags utils", () => {
+    describe("shouldHidePPExperience", () => {
+        it("should not hide pixel perfect experience for free/growth user", () => {
+            const shouldHidePixelperfect = shouldHidePPExperience({
+                enablePixelPerfectExperience: true,
+                platformEdition: "growth",
+                hidePixelPerfectExperience: false,
+            });
+            expect(shouldHidePixelperfect).toBe(false);
+        });
+
+        it("should hide pixel perfect experience for free/growth user when enablePixelPerfectExperience is false", () => {
+            const shouldHidePixelperfect = shouldHidePPExperience({
+                enablePixelPerfectExperience: false,
+                platformEdition: "growth",
+                hidePixelPerfectExperience: false,
+            });
+            expect(shouldHidePixelperfect).toBe(true);
+        });
+
+        it("should hide pixel perfect experience when hidePixelPerfectExperience is true", () => {
+            const shouldHidePixelperfect = shouldHidePPExperience({
+                hidePixelPerfectExperience: true,
+                enablePixelPerfectExperience: false,
+                platformEdition: "growth",
+            });
+            expect(shouldHidePixelperfect).toBe(true);
+        });
+    });
+
+    describe("isFreemiumEdition", () => {
+        it("should return false when user is not free/growth", () => {
+            const isFreemiumCustomer = isFreemiumEdition("enterprise");
+            expect(isFreemiumCustomer).toBe(false);
+        });
+
+        it("should return true when user is free/growth", () => {
+            const isFreemiumCustomer = isFreemiumEdition("growth");
+            expect(isFreemiumCustomer).toBe(true);
+        });
+    });
+
+    describe("shouldEnableNewNavigation", () => {
+        describe("Free user", () => {
+            it("should return false", () => {
+                const enableNewNavigation = shouldEnableNewNavigation({
+                    enableNewNavigationForResponsiveUi: false,
+                    platformEdition: "free",
+                });
+                expect(enableNewNavigation).toBe(false);
+            });
+
+            it("should return true", () => {
+                const enableNewNavigation = shouldEnableNewNavigation({
+                    enableNewNavigationForResponsiveUi: true,
+                    platformEdition: "free",
+                });
+                expect(enableNewNavigation).toBe(true);
+            });
+        });
+
+        describe("Growth user", () => {
+            it("should return false", () => {
```

Comment:
Misleading test name. Should return false when?

Reply:
I have added more details to the name of test cases in this file thanks